packages/flutter/lib/src/material/elevated_button.dart

```diff
@@ -47,6 +47,40 @@ import 'theme_data.dart';
 /// If [onPressed] and [onLongPress] callbacks are null, then the
 /// button will be disabled.
 ///
+/// {@tool dartpad --template=stateless_widget_scaffold}
+///
+/// This sample shows how to render a disabled ElevatedButton, an enabled
```

Comment:
This sample produces an enabled and a disabled ElevatedButton.

Reply:
Thanks @HansMuller 
can you please review it now?
Thanks

Reply:
The style variable should be defined in the build method and it looks like you need to clean up some trailing white space.